crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -257,7 +257,7 @@ def upgrade_controller_nodes
         NodeObject.find(
           "state:crowbar_upgrade AND (roles:database-server OR roles:rabbitmq-server)"
         ).each do |db_node|
-          cmd = "LANG=C crm resource status ms-drbd-{postgresql,rabbitmq}\
+          cmd = "LANG=C crm resource status ms-drbd-postgresql\
```

Comment:
@jsuchome postgresql and rabbitmq could run in different clusters. With this change we would not find a node on the cluster just running rabbitmq. 
It also possible I think to configure one of them to use drbd but the other one for a different type of shared storage.

That being said, it seem this function seems to have some other issues if there are multiple drbd clusters. So maybe we should handle that in a separate PR/Trello card?